freqtrade/constants.py

```diff
@@ -21,6 +22,7 @@
 ORDERTIF_POSSIBILITIES = ['gtc', 'fok', 'ioc']
 AVAILABLE_PAIRLISTS = ['StaticPairList', 'VolumePairList']
 DRY_RUN_WALLET = 999.9
+DEFAULT_DOWNLOAD_TICKER_INTERVALS = '1m 5m'
```

Comment:
isn't the correct syntax 1m,5m for the download script ... ?

Reply:
No, it says "Space separated list."
```
./scripts/download_backtest_data.py --exchange binance -t 1h 1d
```

Reply:
Good point :+1: